django_migration_linter/sql_analyser/analyser.py

```diff
@@ -24,7 +24,10 @@ def get_sql_analyser_class(database_vendor, analyser_string=None):
 def get_sql_analyser_from_string(analyser_string):
     if analyser_string not in ANALYSER_STRING_MAPPING:
         raise ValueError(
-            "Unknown SQL analyser. Known values: {}".format(analyser_string.keys())
+            "Unknown SQL analyser {}. Known values: {}".format(
```

Comment:
```suggestion
            'Unknown SQL analyser "{}". Known values: {}'.format(
```

Reply:
Change applied. Also, changed the quotation marks in the `Unsupported database vendor "{}".` error message.